raster/src/main/scala/geotrellis/raster/io/geotiff/writer/TiffTagFieldValue.scala

```diff
@@ -6,8 +6,10 @@ import geotrellis.raster.io.geotiff.tags.codes._
 import geotrellis.raster.io.geotiff.tags.codes.TagCodes._
 import geotrellis.raster.io.geotiff.tags.codes.TiffFieldType._
 import geotrellis.vector.Extent
-
 import java.nio.ByteOrder
+
+import geotrellis.raster.render.IndexedColorMap
```

Comment:
Ah, I hit "preview" to look at the code, and then failed to hit "Add review comment"...

The order of imports should be as follows:

```scala
// GeoTrellis imports, alphabetical
import geotrellis.raster._
import geotrellis.vector._
import geotrellis.vector.io._

// Third party imports, alphabetical
import org.apache.spark._
import spray.json._

// Java and Scala imports, alphabetical
import java.nio._
import scala.collections.mutable
```